tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -174,9 +174,9 @@ public void testSimpleWebAppWithJSP() throws Exception
                 assertTrue(run2.awaitConsoleLogsFor("Started Server@", 10, TimeUnit.SECONDS));
 
                 startHttpClient();
-                ContentResponse response = client.GET("http://localhost:" + port + "/test/index.jsp");
+                ContentResponse response = client.GET("http://localhost:" + port + "/test/jstl.jsp");
```

Comment:
what about keep both GET on `jsp` and `jstl` because the test name says JSPandJSTL?

Reply:
makes sense